test/test-files/ssl/example.test/with_ca/ecdsa/key.ecdsa.pem

```diff
@@ -3,3 +3,4 @@ MHcCAQEEIMl9lvsiSWUqCQ+cvXdiMApfVhQ/XH8zuiNhFD3JbGv9oAoGCCqGSM49
 AwEHoUQDQgAE9FsAamrKHbgVgIHQgnK+rzo8XqebZCEWGSfzdQvr4P5Hamye19qA
 DhsJdkX+i/15CfcIIhqTICF0XniRU0WecQ==
 -----END EC PRIVATE KEY-----
+someThingsChangedHere
```

Comment:
Mistake? An ASCII file should end with a newline.

Reply:
> Definitely an error, that should not be committed.
> 
> A recent PR added a test that appends to this file, and is meant to perform cleanup after, it seems that failed and accidentally got committed here?

A yes, while switching branches with a test state, this happened, and should have definitely not happened. Sorry!